App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/CreateJob/JobDetailsCard.jsx

```diff
@@ -53,13 +53,14 @@ export class JobDetailsCard extends React.Component {
     }
 
     handleChangeDate(date, name) {
+        var dateMoment = moment(date);
```

Comment:
This might cause an error, specially if the value of **date** is empty or null. Instead, you can verify and confirm first if **date** has a value before doing any code or condition inside this function. Thanks

Reply:
Great catch.   Thanks.